bedrock/firefox/templates/firefox/dashboard-base.html

```diff
@@ -7,7 +7,7 @@
 {% block page_title_prefix %}{% endblock %}
 {% block page_title %}{% endblock %}
 
-{% block site_css %}
+{% block page_css %}
   {% stylesheet 'firefox-dashboard' %}
```

Comment:
Yep, I think leaving `site_css` in `dashboard-base.html` and using `page_css` in `interest-dashboard.html` and `tiles.html` is the way to go.

`{{ super() }}` basically means "insert the content of the parent template's block", meaning you can kind of extend a block instead of straight up overwriting.

So, I think `dashboard-base.html` would look like:

```
{% block site_css %}
  {# brings in the site_css bundle specified in the parent template (firefox/base-resp.html) #}
  {{ super() }}
  {% stylesheet 'firefox-dashboard' %}
{% endblock %}
```

Each child page would then have a basic `page_css` block, e.g.

```
{% block page_css %}
  {% stylesheet 'firefox-interest-dashboard' %}
{% endblock %}
```